features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^the following dishes exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:dish, hash)
+  end
+end
+
+When(/^I am on the landing page$/) do
```

Comment:
Is the `dishes_path` really the `landing page`? It seems to conflict with the story the other team is working on.